src/esp/physics/bullet/BulletRigidObject.h

```diff
@@ -36,8 +36,7 @@ namespace physics {
  * state with internal btRigidBody states
  */
 class BulletRigidObject : public BulletBase,
-                          public RigidObject,
-                          public Magnum::BulletIntegration::MotionState {
```

Comment:
Same here. RigidObject state getters called after setters and before simulation step may need to manually update the node states since MotionState is not doing this automatically.